### PR TITLE
Add a note for tidb_restricted_read_only

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/9822
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by SunRunAway — APPROVED** [review]

File: system-variables.md [hunk 1]
```diff
@@ -609,6 +609,10 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_restricted_read_only` <span class="version-mark">从 v5.2.0 版本开始引入</span>
 
+> **警告：**
+>
+> 在 v5.3.1 或 v5.4.1 之前的 TiDB 版本中，该变量存在问题，使用该变量可能导致非预期的结果。如需使用该变量，请确保你的 TiDB 版本不低于 v5.3.1 或 v5.4.1。
```
Comment (TomShawn) [inline system-variables.md:614]: 英文是 "later than v5.3.1 or v5.4.1."
```suggestion
> 在 v5.3.1 或 v5.4.1 之前的 TiDB 版本中，该变量存在问题，使用该变量可能导致非预期的结果。如需使用该变量，请确保你的 TiDB 版本高于 v5.3.1 或 v5.4.1。
```

File: system-variables.md [hunk 2]
```diff
@@ -609,6 +609,10 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
 
 ### `tidb_restricted_read_only` <span class="version-mark">从 v5.2.0 版本开始引入</span>
 
+> **警告：**
+>
+> 在 v5.3.1 或 v5.4.1 之前的 TiDB 版本中，该变量存在问题，使用该变量可能导致非预期的结果。如需使用该变量，请确保你的 TiDB 版本高于 v5.3.1 或 v5.4.1。
```
Comment (TomShawn) [inline system-variables.md:614]:
```suggestion
> 在 v5.3.1 或 v5.4.1 之前的 TiDB 版本中，使用该变量可能导致非预期的结果。如需使用该变量，请确保你的 TiDB 版本高于 v5.3.1 或 v5.4.1。
```

**Review by TomShawn — APPROVED** [review]

**Pull request merged**